pkgs/tools/misc/kb/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "kb";
+  version = "0.1.7";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "gnebbia";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hjvf1ycwf4f8rnrshj30ij56mkji2ic3ibpschyvng5jslh1h9b";
```

```suggestion
    hash = "...";
```

Use `nix hash to-sri --type sha256 0hjvf1ycwf4f8rnrshj30ij56mkji2ic3ibpschyvng5jslh1h9b` to find out the SRI hash.